Makefile

```
@@ -129,7 +129,7 @@ gosec: ## Run gosec against code.
 
 .PHONY: lint
 lint: ## Run lint against code.
-	docker run --rm -w /workdir -v $(PWD):/workdir golangci/golangci-lint:latest golangci-lint run -c .golangci.yml --fix
+	docker run --rm -w /workdir -v $(PWD):/workdir golangci/golangci-lint:v2.1.5 golangci-lint run -c .golangci.yml --fix
```

```suggestion
GOLANGCI_LINT_VERSION ?= v2.1.5
	docker run --rm -w /workdir -v $(PWD):/workdir golangci/golangci-lint:$(GOLANGCI_LINT_VERSION) golangci-lint run -c .golangci.yml --fix
```

I would make the golangci-lint version a variable here instead of hardcoding

---

updated 😄